src/1.0.0/reference/sql.md

```diff
@@ -18,46 +18,50 @@
 ### Creating tables
 
 ```
-CREATE TABLE table1 (id INTEGER, PRIMARY KEY id);
-CREATE TABLE table1 (id INTEGER, ts INTEGER, title VARCHAR, active BOOLEAN, payload BLOB, PRIMARY KEY id);
-CREATE TABLE IF NOT EXISTS table1 (id INTEGER, PRIMARY KEY id);
+CREATE TABLE IF NOT EXISTS customers (id INTEGER, customer_name VARCHAR, email VARCHAR, address VARCHAR, city VARCHAR, ip VARCHAR, country VARCHAR, age INTEGER, active BOOLEAN, PRIMARY KEY id);
```

Comment:
Once immudb PR `https://github.com/codenotary/immudb/pull/900` gets merged, variable sized columns i.e. VARCHAR and BLOB will require a maximum size to be part of an index e.g. `customer_name VARCHAR[50]`

Reply:
@jeroiraz do you think it's a good idea to update further that page to avoid letting people add variable sized columns and then later they could be stuck not being able to create a new index?